robottelo/ui/base.py

```diff
@@ -163,14 +202,9 @@ def delete_entity(self, name, really, name_locator, del_locator,
         # Make sure that element is really removed from UI. It is necessary to
         # verify that fact few times as sometimes 1 second is not enough for
         # element to be actually deleted from DB
+        self.is_entity_delete = True
         for _ in range(3):
```

Comment:
Logically, we do not need it as per overall flow, there is a chance of 0.000001% that we will need to re-use search after we do delete and all necessary verification of entity existence. If we get a failure on any step - we will fail test case itself and proceed to the next (class variable value will be erased).

But, if you think that we need it - just tell and I will add try-finally block